examples/distributed.jl

```diff
@@ -0,0 +1,82 @@
+using Distributed
+
+addprocs(2)
+# Use Distributed.jl to do a control channel setup
+# TODO:
+#  - shutdown
+#  - Using one-sided to avoid having to figure out bind_addr
+
+@everywhere using UCX
+
+@everywhere begin
+function start(callback)
+    ctx = UCX.UCXContext()
+    worker = UCX.Worker(ctx)
+
+    global UCX_WORKER = worker
+    atexit() do
+        close(worker)
+    end
+    addr = UCX.UCXAddress(worker.worker)
+    let worker = worker
+        addr = addr
+        Threads.@spawn begin
+            try
+                callback(worker, UCX.UCXEndpoint(worker.worker, addr))
+            catch err
+                showerror(stderr, err, catch_backtrace())
+            end
+        end
+    end
+
+    @async begin
+        GC.@preserve listener begin
+            while isopen(worker.timer)
+                wait(worker)
+            end
+        end
+        close(worker)
+    end
+
+    GC.@preserve addr begin
+        ptr = Base.unsafe_convert(Ptr{UInt8}, addr.handle)
+        addr_buf = Base.unsafe_wrap(Array, ptr, addr.len; own=false)
+        bind_addr = similar(addr_buf)
+        copyto!(bind_addr, addr_buf)
+    end
+
+    return bind_addr
+end
+
+function process(worker::UCX.Worker, ep::UCX.UCXEndpoint)
```

Comment:
Print a message so that we know what this callback is supposed to be able to do, or at least add a comment indicating what a user could put here